scripts/release.js

```diff
@@ -156,6 +156,15 @@ logger.warn(`
 
 shouldContinuePrompt();
 
+// Creates a new React test app, install the component app as dependency and build the test app to ensure no errors or build issues
+process.chdir(path.join(__dirname, '../..'));
+spawnOrFail('npx', ['create-react-app test-app']);
```

Comment:
lets call this something else - dependency-check-app

Reply:
updated. thanks!